virttest/staging/backports/simplejson/encoder.py

```diff
@@ -4,6 +4,12 @@
 from decimal import Decimal
 
 
+try:
+    long
+except NameError:
+    long = int
```

Comment:
Seems the packages under directory `backports` are provided for old pythons (<=2.5), so we do not have to modify them if not necessary.

Reply:
@luckyh "vt-bootstrap" will not scan the backports directory? And I think this change will not affect old pythons